src/components/calcite-action/calcite-action.e2e.ts

```diff
@@ -133,40 +133,4 @@ describe("calcite-action", () => {
 
     expect(clickSpy).toHaveReceivedEventTimes(0);
   });
-
-  it("should emit 'calciteActionClick' event", async () => {
```

Comment:
Should these be removed when `calciteActionClick` is dropped?

Reply:
I think its safe enough to remove now.